contrib/python-emoji/template.py

```diff
@@ -0,0 +1,21 @@
+pkgname = "python-emoji"
+pkgver = "2.12.1"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+    "python-wheel",
+]
+checkdepends = ["python-pytest", "python-typing_extensions"]
```

Comment:
typing_extensions is an actual dependency and not checkdepends (that you should patch out)